scripts/tasks/webpack-resources.js

```diff
@@ -9,20 +9,22 @@ console.log(`Webpack version: ${webpackVersion}`);
 module.exports = {
   webpack,
 
-  createConfig(packageName, isProduction, customConfig, onlyProduction) {
+  createConfig(packageName, isProduction, customConfig, onlyProduction, excludeSourceMaps) {
```

Comment:
since we don't have typing info, maybe we should move this into an options object?

Reply:
yes, just a bigger change, trying to simplify. I can take a look in options bag and fixing all the calls